docs/platform/tutorials/train-and-deploy-ml/train-ml.md

```diff
@@ -0,0 +1,256 @@
+# Train your ML model
+
+In this part you learn how to train an ML model in Jupyter Notebook, using the data you imported in the [previous part](./import-data.md). 
+
+You write code to train your model on this data. You save your model to a Pickle file, which you then deploy in the [next part](./deploy-ml.md) of this tutorial. 
+
+![](./images/train.png)
+
+This is just one approach that you might use if you are already familiar with Jupyter Notebook. You might also train your ML model directly in Quix using live data, or data played back using the [replay feature](../../how-to/replay.md).
+
+## Install the required libraries
+
+You now need to install some Python libraries on your system. This is the system where you are running Jupyter Notebook. If you do not have Jupyter Notebook installed, please refer to the [prerequisites](./index.md#prerequisites) for this tutorial.
+
+The following libraries are required to create the ML model and visualize your data. To install the libraries enter the following commands into a terminal on the system where you are running Jupyter Notebook:
+
+``` shell
+python3 -m pip install seaborn
```

Comment:
I need to remove `python3 -m` for mine to work. Im on windows

Reply:
Removed. Just pip commands used now.